qtservice/src/qtservice_unix.cpp

```diff
@@ -240,7 +296,12 @@ bool QtServiceController::sendCommand(int code)
 
 bool QtServiceController::isInstalled() const
 {
-    QSettings settings(QSettings::SystemScope, "QtSoftware");
+
+    if (isSystemD()) {
+        return QFileInfo::exists(systemDPath() + serviceName() + ".service");
+    }
+
+    auto &settings = getSettings(serviceName());
```

Comment:
There are not modify. use QSettings settings(QSettings::SystemScope, "QtSoftware");